matter_server/common/models/message.py

```diff
@@ -0,0 +1,73 @@
+"""Models for the Websocket messages."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum, IntEnum
+from typing import Any, Type, Union
+
+from ..helpers.util import dataclass_from_dict
+from .events import EventType
+
+
+@dataclass
+class CommandMessage:
+    """Model for a Message holding a command from server to client or client to server."""
+
+    messageId: str
```

Comment:
Can we use snake_case attribute names instead?

Reply:
sorry, leftover from the previous implementation, which in turn came from wave js node js implementation.
I'm in favor of adjusting it all to snake case